src/server/data_store/BUILD

```diff
@@ -0,0 +1,23 @@
+package(
```

Comment:
nit: This is not consistent with our other BUILD style.

I feel like we should install buildifier (https://github.com/bazelbuild/buildtools/tree/master/buildifier)

However, we can fix this later in a formating only PR